src/frequenz/sdk/timeseries/logical_meter/_logical_meter.py

```diff
@@ -62,16 +74,15 @@ class LogicalMeter:
         )
 
         # Get a receiver for a builtin formula
-        grid_power_recv = logical_meter.grid_power()
+        grid_power_recv = logical_meter.grid_power
```

Comment:
This is the type of error that `pylint` can't detect :(

`mypy` should though.

```suggestion
        grid_power_recv = logical_meter.grid_power.new_receiver()
```